install/ansible/tasks/bootstrap_ubuntu.yaml

```diff
@@ -0,0 +1,15 @@
+---
+- name: "Install required packages on Ubuntu"
+  package:
+    name: "{{ item }}"
+    state: latest
+  with_items:
+      - unzip
+      - tar
+      - apt-transport-https
+      - btrfs-tools
+      - libapparmor-dev
```

Comment:
Do we need `-dev` library? And same question for the other `-dev` library.

Reply:
I saw that in our readme so I added this here. Let me know if we need to add a different package